src/electron/auto-update/auto-updater-client.ts

```diff
@@ -1,8 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import log from 'electron-log';
 
 export class AutoUpdaterClient {
-    constructor(private readonly autoUpdater) {}
+    constructor(private readonly autoUpdater) {
+        log.transports.file.level = 'info';
+        autoUpdater.logger = log;
```

Comment:
Is there any way to test this is called? or at least that `logger` is indeed `log` ???
I'm surprise no test file has been changed as part of the changes on this constructor.